driver/driver.go

```diff
@@ -457,5 +463,36 @@ func NewDriver(ctx context.Context, cfg *config.Driver) (Driver, error) {
 		return nil, err
 	}
 
-	return &driver{driverWithOptions: drv}, nil
+	var wg sync.WaitGroup
```

Comment:
Can we move it to separate function?
Something like `wd, ch := startHealthPingLoop(cfg.PingInterval, drv)`

Reply:
fixed.